go.mod

```diff
@@ -1,6 +1,6 @@
 module github.com/gobuffalo/clara/v2
 
-go 1.16
+go 1.17
```

Comment:
This is what I don't want to do for now because:

- the policy supporting the last two versions is a strategy that we cannot support all historical versions and this means we will make the module compatible with at least those two versions. We will support users if they have an issue with those supported versions proactively.
- however, setting this as `go 1.17` means the module will not be able to be used with the old version of go even though the code itself has no version-specific things within it by forcing the minimum version as 1.17.

Could you please remove this, `go mod tidy` again, and commit so the module can be still useful for those who cannot upgrade their environment yet?

Reply:
Update for future reference: the second line is not literally true but I wanted to say that it can mislead version dependency and changes.